com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/font/Fontc.java

```diff
@@ -672,7 +672,7 @@ else if (channelCount > 1)
                 .setCharacter(glyph.c)
                 .setWidth(glyph.width + (glyph.width > 0 ? padding * 2 : 0))
                 .setAdvance(glyph.advance)
-                .setLeftBearing(glyph.leftBearing - padding)
```

Comment:
given the cursor stepping in font_renderer.cpp, I feel this is an old bug, that we haven't seen it's most notable if you put values like "30" into the shadow blur.

![bild](https://user-images.githubusercontent.com/1349334/89782177-1026dd00-db15-11ea-8d5d-db1a6df0d6bc.png)
GIven this image, I feel that if the `advance` isn't altered, the `left bearing` shouldn't be either?